hosting_docs/source/installation/migration/1-migrating-project.rst

```diff
@@ -117,25 +117,15 @@ will be unable to submit forms or sync with the server.
 3. Prepare the new environment to be populated
 ----------------------------------------------
 
+* Setup a new environment by following :ref:`deploy-commcarehq`
+* Do a commcare-hq deploy using :ref:`operations/2-deploys:Deploying CommCare HQ code changes`
+* And now proceed to step 4.
```

Comment:
Very nit and very non blocking of nature:
```suggestion
* Proceed to step 4.
```
The fact that it's listed in the next bullet point implies the "And now" (I have a weird thing about these types of stuff)

Reply:
👍